bucket/snaptimer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.1",
+    "description": "SnapTimer - Free Windows Countdown Timer",
+    "homepage": "https://dan.hersam.com/software/snaptimer/index.php",
+    "license": "Freeware",
+    "url": "https://cdn.hersam.com/software/snaptimer/SnapTimer.zip",
+    "hash": "227df1414eefda1d98249d6e96465266a9b6517e29808e0eb57f3eca8875d23b",
+    "extract_dir": "SnapTimer",
+    "bin": "SnapTimer.exe",
```

Comment:
What are available command-line arguments?

If none, remove bin property

Reply:
* Accepts time as command line argument, so you can create preset timer shortcuts or run the timer from a launcher (FARR, Launchy).